test/test_optimistic_txn.rs

```diff
@@ -0,0 +1,131 @@
+use exonum_rocksdb::{Options, WriteOptions, IteratorMode};
+use exonum_rocksdb::optimistic_txn_db::{OptimisticTransactionDB, OptimisticTransactionOptions};
+use tempdir::TempDir;
+
+#[test]
+fn test_optimistictransactiondb_creation_and_destroy() {
+    let temp_dir = TempDir::new("transaction_db_1").unwrap();
+    let path = temp_dir.path();
+    let _ = OptimisticTransactionDB::open_default(path);
+    assert!(OptimisticTransactionDB::destroy(&Options::default(), path).is_ok());
+}
+
+#[test]
+fn test_optimistictransactiondb_transaction() {
+    let temp_dir = TempDir::new("transaction_db_2").unwrap();
+    let path = temp_dir.path();
+    let db = OptimisticTransactionDB::open_default(path).unwrap();
+    let mut w_opts = WriteOptions::default();
+    w_opts.set_sync(true);
+    let txn_opts = OptimisticTransactionOptions::default();
+
+    {
+        let txn1 = db.transaction_begin(&w_opts, &txn_opts);
+        let txn2 = db.transaction_begin(&w_opts, &txn_opts);
+
+        assert!(txn1.put(b"a", b"1").is_ok());
```

Comment:
Perhaps it will be convenient to declare constants for transactions and values. Similar to what we have in sandbox tests:
```rust
static FIRST_KEY: &str = "a";
static FIRST_VALUE: &str = "1";
...
```